lib/class-importer.php

```diff
@@ -596,25 +605,55 @@ public function import_item( array $data, $parent_post_id = 0, $import_ignored =
 
 			switch ( $post_data['post_type'] ) {
 				case $this->post_type_class:
-					$this->errors[] = "\t" . sprintf( 'Problem inserting/updating post for class "%1$s"', $data['name'], $post_id->get_error_message() );
+					$this->errors[] = "\t" . sprintf( 'Problem inserting/updating post for class "%1$s"', $ns_name, $post_id->get_error_message() );
 					break;
 
 				case $this->post_type_method:
-					$this->errors[] = "\t\t" . sprintf( 'Problem inserting/updating post for method "%1$s"', $data['name'], $post_id->get_error_message() );
+					$this->errors[] = "\t\t" . sprintf( 'Problem inserting/updating post for method "%1$s"', $ns_name, $post_id->get_error_message() );
 					break;
 
 				case $this->post_type_hook:
 					$indent = ( $parent_post_id ) ? "\t\t" : "\t";
-					$this->errors[] = $indent . sprintf( 'Problem inserting/updating post for hook "%1$s"', $data['name'], $post_id->get_error_message() );
+					$this->errors[] = $indent . sprintf( 'Problem inserting/updating post for hook "%1$s"', $ns_name, $post_id->get_error_message() );
 					break;
 
 				default:
-					$this->errors[] = "\t" . sprintf( 'Problem inserting/updating post for function "%1$s"', $data['name'], $post_id->get_error_message() );
+					$this->errors[] = "\t" . sprintf( 'Problem inserting/updating post for function "%1$s"', $ns_name, $post_id->get_error_message() );
 			}
 
 			return false;
 		}
 
+		$namespaces = ( ! empty( $data['namespace'] ) ) ? explode( '\\', $data['namespace'] ) : false;
```

Comment:
We were discussing doing this as a separate PR to start breaking this whole function up in to smaller pieces. Looking at the larger context, I was remembering a different set of functions when discussing it, and agree with the comment. We can do this here, but we should still probably circle back and break out more of this import function into smaller pieces.